app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CartItemsController < ApplicationController
+  before_action :current_cart
+
+  def index
+    @cart_items = @current_cart.cart_items.all
+  end
+
+  def create
+    chosen_food = Food.find(params[:food_id])
+    current_cart = @current_cart
```

Comment:
Why you are assigning instance variable to local variable can you please use instance everywhere